app/models/actions/propose_evidence_item.rb

```diff
@@ -49,7 +49,7 @@ def get_disease(params)
       if params[:noDoid]
         proposed_name = Disease.capitalize_name(params[:disease_name])
         Disease.where(display_name: proposed_name, name: proposed_name).first_or_create
-      else
+      elsif params[:disease]
```

Comment:
This isn't a problem for this PR to fix, more a comment on our general need for better validation around EID state, but now there's no distinction between submitting a valid EID that just doesn't have a disease because of its type, and submitting an EID that _should_ have a disease and just not supplying the parameter.

For V2 I think we should just rework this logic with more formal validation across the board.